test/CMakeLists.txt

```diff
@@ -2,5 +2,4 @@
 add_executable(testregex testregex.cpp)
 add_test(testregexcorrect testregex)
 
-
 # correctness testing of checks
```

Comment:
Should **check_clang_tidy.py** be used? If so, I should add an error code return statement to the script as you mentioned in your comment to the script. And where is it specified which script will be used for the test?